### PR TITLE
Merge 1.2.x

### LINKED ISSUE #4025: Duplicate values in CORS header when using socket.io
**Rasa version**: 1.1.6

**Python version**: 3.6

**Operating system** (windows, osx, ...): osx

**Issue**:
- the http server creates duplicate values in the CORS header `Access-Control-Allow-Origin` when used with socket.io as an input channel
- issue seems to be related to using sanic in combination with python-socketio, and is known: https://github.com/miguelgrinberg/python-socketio/issues/205
- reported here: https://forum.rasa.com/t/cors-multiple-values-in-response-header/12749
- possible solution described here: https://github.com/miguelgrinberg/python-socketio/issues/205#issuecomment-493769183

**Error (including full traceback)**:
```
Access to XMLHttpRequest from origin has been blocked by CORS policy: The ‘Access-Control-Allow-Origin’ header contains multiple values, but only one is allowed.
```

### PR DESCRIPTION
**Pull request by akelad**
**Proposed changes**:
- merge latest 1.2.x changes into master
- fixes #4025

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

**Comment by wochinge** [comment]
Should it not be from `1.2.x` instead of from `socketio-fix`?

**Comment by akelad** [comment]
yeah that's the latest version of the branch though. If I do it from 1.2.x then I'll be merging master into that to resolve conflicts, which we do not want ;)

File: CHANGELOG.rst [hunk 1]
```diff
@@ -41,6 +41,33 @@ Fixed
...
 
+Fixed
+-----
+
+
+[1.2.4] - 2019-08-23
+^^^^^^^^^^^^^^^^^^^^
+
+Added
+-----
```
Comment (wochinge) [inline CHANGELOG.rst]: let's remove all these empty ones and just keep the `Fixed`

Comment (akelad) [reply CHANGELOG.rst]: oooops

Comment (akelad) [reply CHANGELOG.rst]: should've done that in the released version 😬

**Pull request closed without merging**